integreat_cms/cms/utils/external_calendar_utils.py

```diff
@@ -35,6 +35,7 @@ class IcalEventData:
     categories: list[str]
     external_calendar_id: int
 
+    # pylint: disable=too-many-locals
```

Comment:
Apparently we have been doing this wrong the entire time :see_no_evil: 
More context: https://github.com/digitalfabrik/integreat-cms/pull/3075#discussion_r1801247908

Github won't let me make a suggestion, but the fix would be to move this line to the first line of the function